app/src/main/java/com/quran/labs/androidquran/ui/util/TranslationsSpinnerAdapter.java

```diff
@@ -36,6 +37,9 @@ public TranslationsSpinnerAdapter(Context context,
     // and the String[] constructor makes a new (immutable) List with the items of the array.
     super(context, resource, new ArrayList<>());
     this.layoutInflater = LayoutInflater.from(context);
+    List<String> transList = new ArrayList<String>(Arrays.asList(translationNames));
+    transList.add("More Translations");
+    translationNames = transList.toArray(new String[transList.size()]);
```

Comment:
I did this initially as `translationNames` is also used in `addAll(translationNames);`, should "More translations" not be a part of the underlying array?